UI/Selenium/Steps/HearingListSteps.cs

```diff
@@ -46,9 +46,9 @@ public void ProceedToWaitingRoom(string participant, string caseNumber)
             Driver = GetDriver(participant, _scenarioContext);
             _scenarioContext["driver"] = Driver;
             Driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(Config.DefaultElementWait);
-            var elements = Driver.FindElements(JudgeHearingListPage.HealingListRow);
+            ExtensionMethods.FindElementWithWait(Driver, ParticipantHearingListPage.SelectButton(_hearing.Case.CaseNumber), _scenarioContext, TimeSpan.FromSeconds(Config.DefaultElementWait));
             Driver.FindElement(ParticipantHearingListPage.SelectButton(_hearing.Case.CaseNumber)).Click();
-            if (!participant.ToLower().Contains("judge"))
+            if (!(participant.ToLower().Contains("judge") || participant.ToLower().Contains("panel")))
```

Comment:
This might be wrong. There are also Judicial office holders and wingers to include when you have more that one type of judge.

Reply:
The current scenario is for judge and panel member. We can add more to the list as and when we deal with different types of judge